content/v2.0/query-data/flux/manipulate-timestamps.md

```diff
@@ -26,13 +28,21 @@ Use Flux to process and manipulate timestamps to suit your needs.
 - [Convert timestamp format](#convert-timestamp-format)
 - [Time-related Flux functions](#time-related-flux-functions)
 
+---
+
 If you're just getting started with Flux queries, check out the following:
 
 - [Get started with Flux](/v2.0/query-data/get-started/) for a conceptual overview of Flux and parts of a Flux query.
 - [Execute queries](/v2.0/query-data/execute-queries/) to discover a variety of ways to run your queries.
 
+---
+
 ## Convert timestamp format
 
+- [Convert unix nanosecond timestamp to RFC3339](#convert-unix-nanosecond-timestamp-to-rfc3339)
+- [Convert RFC3339 to nanosecond unix timestamp](#convert-rfc3339-to-nanosecond-unix-timestamp)
+- [Calculate the duration between two timestamps](#calculate-the-duration-between-two-timestamps)
```

Comment:
Is "Calculate the duration between two timestamps" supposed to be a subtopic of "Convert timestamp format"?

Reply:
Maybe drop "Time-related Flux functions group" heading and reorg as task-based headers:

- Convert timestamp format:
   - Unix to...
   - RFC3339 to...
- Retrieve current time:
   - UTC
   - System
- Work with timestamp durations:
    - Calculate duration between
    - Add duration to
    - Subtract a duration from